receiver/awsecscontainermetricsreceiver/receiver_test.go

```diff
@@ -17,28 +17,38 @@ package awsecscontainermetricsreceiver
 import (
 	"context"
 	"errors"
+	"fmt"
 	"io/ioutil"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/component/componenttest"
 	"go.opentelemetry.io/collector/consumer/consumertest"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/ecsutil"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsecscontainermetricsreceiver/internal/awsecscontainermetrics"
 )
 
 type fakeRestClient struct {
 }
 
-func (f fakeRestClient) EndpointResponse() ([]byte, []byte, error) {
-	taskStats, err := ioutil.ReadFile("testdata/task_stats.json")
-	if err != nil {
-		return nil, nil, err
+func (f fakeRestClient) GetResponse(path string) ([]byte, error) {
```

Comment:
I suggest using `embed` instead of reading the files each time here since they would be in memory already and would help speed up tests

Reply:
I think this is the standard method of handling `testdata` files in the project and don't believe embed is used anywhere else.  If it were part of the application I can see that being helpful though.

Reply:
Yeah, that is reasonable :)

Reply:
I actually found this necessary after consolidating a test helper and am now a convert.  Thanks for the suggestion.